atom/browser/api/lib/web-contents.coffee

```diff
@@ -79,6 +93,39 @@ wrapWebContents = (webContents) ->
 
     @_printToPDF printingSetting, callback
 
+  webContents.session.clearStorageData = (options, callback) ->
+    if not callback? and typeof options is 'function'
```

Comment:
The `callback` parameter should be required, we only allow optional callback when there is a synchronous version of the API.